ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -104,6 +104,7 @@ conformance_test(
             "TLSAtLeastOnePointTwoIT",
             "CommandDeduplicationPeriodValidationIT:OffsetPruned",  # requires pruning not available in canton community
             "DeeplyNestedValueIT",  # FIXME: Too deeply nested values flake with a time out (half of the time)
+            "TransactionServiceVisibilityIT:TXTreeChildOrder",  # FIXME: Temporarily disabled: test enhanced testing childEventId order
```

Comment:
FYI: temporarily disabling in canton (this will be corrected after canton version turn-around)